src/components/utils/errors.js

```diff
@@ -0,0 +1,8 @@
+export const errors = {
```

Comment:
you might want to create a module called strings, but I would keep it outside of any `utils` naming.